glide.lock

```diff
@@ -233,7 +233,7 @@ imports:
 - name: github.com/uber-go/atomic
   version: 8474b86a5a6f79c443ce4b2992817ff32cf208b8
 - name: github.com/uber/jaeger-client-go
-  version: b043381d944715b469fd6b37addfd30145ca1758
+  version: 1a782e2da844727691fef1757c72eb190c2909f0
```

Comment:
Can you update the verson in glide.yml?

Reply:
Yes I did, but still error from ingester.